giraffe/src/types/index.ts

```diff
@@ -255,6 +255,7 @@ export enum LayerTypes {
   Mosaic = 'mosaic',
   Table = 'table',
   Geo = 'geo',
+  SimpleTable = 'simple table',
```

Comment:
Not a big deal but this should have waited until there was code that uses this definition.